lib/postcssPlugin.js

```diff
@@ -1,9 +1,9 @@
+// @ts-nocheck
```

Comment:
Types from PostCSS don't have `postcss.plugin` type anymore.

This file should be rebuild with visitors API anyway, so ignoring the problem for now.

Reply:
Sounds good.

I believe this is how CSSNano pulls together its PostCSS@8 plugin pack:

https://github.com/cssnano/cssnano/blob/b60f54bedafe3781ff58f0888ab45ff5c56aee09/packages/cssnano/src/index.js#L91-L132